### DISCUSSION TITLE
Add attribute-parameter to badges

### DISCUSSION THREAD

**Discussion by KixAss** [home-assistant/frontend#18683] [Dashboard]
It would be nice to add an attribute-parameter to badges, so you can set what info to show.

For example: I have a badge with an entity of a climate (dual thermostat). Currently it shows the mode (off, heating, etc). I'd like to see the temperature.

And yes: i could create a template-sensor, but i want to edit the temperature when I click on the badge.

**Comment by ildar170975** [comment]
Vote for this.
Badges are used in a floorplan by many users.
A support to show attributes is highly needed.
